addons/web/controllers/home.py

```diff
@@ -147,10 +148,17 @@ def switch_to_admin(self):
         return request.redirect(self._login_redirect(uid))
 
     @http.route('/web/health', type='http', auth='none', save_session=False)
-    def health(self):
-        data = json.dumps({
-            'status': 'pass',
-        })
+    def health(self, db_server_status=False):
+        health_info = {"status": "pass"}
+        status = 200
+        if db_server_status:
+            try:
+                odoo.sql_db.db_connect('postgres').cursor().close()
+                health_info["db_server_status"] = True
+            except psycopg2.Error:
+                health_info["db_server_status"] = False
```

Comment:
```suggestion
                health_info["db_server_status"] = False
                health_info["status"] = "fail"
```

Reply:
To me we can't do that as it is a change in the behavior. This can be made in master.

Reply:
In other words today ´status´ reflects the http server state.

Reply:
Mmmh after having thought about it as this won't be triggered if the argument is not passed it is arguable indeed 😁